e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,158 @@
+        getJpgPost().invoke('height').as('initialJpgHeight');
+        getAttachmentPost().invoke('height').as('initialAttachmentHeight');
+        getInlineImgPost().invoke('height').as('initialInlineImgHeight');
+        getLastTextPost().invoke('height').as('initialLastPostHeight');
+
+        // # Switch the account settings for the test user to enable Fixed width center
+        cy.toAccountSettingsModal();
+        cy.get('#accountSettingsModal').should('be.visible').within(() => {
+            cy.findByText('Display', {timeout: timeouts.ONE_MIN}).click();
+            cy.findByText('Channel Display').click();
+            cy.findByLabelText('Fixed width, centered').click();
+            cy.findByText('Save').click();
+            cy.get('#accountSettingsHeader > .close').click();
+        });
+
+        // # Browse to Channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // * Verify All posts are displayed correctly
+        cy.findAllByTestId('postContent').should('have.length', '9').and('have.class', 'post__content center');
+
+        // * Verify there is no scroll pop
+        cy.get('#post-list').should('exist').within(() => {
+            cy.get('@initialUserNameHeight').then((originalHeight) => {
+                getUserNameTitle().eq(0).should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialFirstPostHeight').then((originalHeight) => {
+                getFirstTextPost().should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialLastPostHeight').then((originalHeight) => {
+                getLastTextPost().should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialMp3Height').then((originalHeight) => {
+                getMp3Post().should('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialMpgHeight').then((originalHeight) => {
+                getMpgPost().should('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialGifHeight').then((originalHeight) => {
+                getGifPost().should('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialJpgHeight').then((originalHeight) => {
+                getJpgPost().should('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialInlineImgHeight').then((originalHeight) => {
+                getInlineImgPost().should('have.css', 'height', (originalHeight + 2) + 'px');
+            });
+            cy.get('@initialAttachmentHeight').then((originalHeight) => {
+                getAttachmentPost().should('have.css', 'height', (originalHeight + 2) + 'px');
+            });
+        });
+    });
+
+    const getUserNameTitle = () => {
+        return cy.findAllByLabelText('sysadmin');
+    };
+
+    const getMp3Post = () => {
```

Comment:
Could you please add a comment to explain what this section is doing?

Reply:
:heavy_check_mark: